spec/dir_class_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'spec_helper'
+
+describe "Ext3::Dir class methods" do
+  def cassette_path
+    "spec/cassettes/dir_class.yml"
+  end
+
+  describe ".[]" do
+    it "should return empty array when in a nonexistent directory" do
+      VirtFS.cwd = "/not_a_dir" # bypass existence checks.
+      expect(VirtFS::VDir["*"]).to match_array([])
+    end
+
+    it "should enumerate the same file names as the standard Dir.glob - simple glob" do
```

Comment:
@roliveri not fully following, these tests are testing access to the mounted filesystem (the actual mounting occurs in 'before' hooks in spec_helper.rb).